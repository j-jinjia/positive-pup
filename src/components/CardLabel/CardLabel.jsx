import "./CardLabel.scss";

const CardLabel = ({ name }) => {
  return (
    <div className="label">
      <span className="label__text">{name}</span>
    </div>
  );
};

export default CardLabel;
