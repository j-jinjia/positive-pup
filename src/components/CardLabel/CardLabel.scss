@use "../../styles/variables" as *;

.card-label {
  background-color: $colour-secondary-background;
  padding: 0.5rem;
  padding-left: 0.8rem;
  padding-right: 0.8rem;
  border-radius: 40px;
  font-size: $font-size-tiny;
  &__text {
    color: $colour-font-label;
  }
}
