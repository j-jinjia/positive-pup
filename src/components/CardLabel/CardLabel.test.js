import { customRender } from "../../utils/testUtils";
import CardLabel from "./CardLabel"

it("should match CardLabel component snapshot", () => {
  const { container } = customRender(<CardLabel name="Online" />);

  expect(container).toMatchSnapshot();
});
