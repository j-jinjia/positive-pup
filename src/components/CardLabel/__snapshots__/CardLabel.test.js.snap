// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match CardLabel component snapshot 1`] = `
<div>
  <div
    class="card-label"
  >
    <span
      class="card-label__text"
    >
      Online
    </span>
  </div>
</div>
`;
